modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/doc/JavaDocReader.java

```diff
@@ -109,6 +113,24 @@ private static String parseParameter(String token, Map<String, StringBuilder> pa
     return currentParamName;
   }
 
+  private static String clean(String text) {
+    return removeHTML(stripTags(text));
+  }
+
+  /**
+   * Removes all HTML from the text and converts anchor to asciidoc valid links. For example: <a href="url">Inner</a> becomes url[Inner].
+   */
+  private static String removeHTML(String text) {
+    String textWithAnchors = Jsoup.clean(text,
+                                         new Whitelist().none().addTags("a").addAttributes("a", "href")
+                                             .addProtocols("a", "href", "http", "https"));
+    Document htmlDoc = Jsoup.parseBodyFragment(textWithAnchors);
+    for (org.jsoup.nodes.Element anchor : htmlDoc.select("a")) {
+      anchor.html(anchor.attr("href") + "[" + anchor.html() + "]");
+    }
+    return Jsoup.clean(htmlDoc.html(), new Whitelist().none());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Change this instance-reference to a static reference. ([squid:S2209](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2209))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7672&issues=AWjew_0QotIXnmsUUoY6&open=AWjew_0QotIXnmsUUoY6)

[ISSUE_KEY]: AWjew_0QotIXnmsUUoY6